modules/ROOT/pages/choice-router-concept.adoc

```diff
@@ -85,6 +87,50 @@ http://www.mulesoft.org/schema/mule/http http://www.mulesoft.org/schema/mule/htt
 </mule>
 ----
 
+== Adding Additional Route Options
+
+By default, the Choice router has one route option and a default route. To add additional routes to the Choice router, follow these steps:
+
+. From *Mule Palette* in Anypoint Studio, click and hold any component.
+. Drag the component inside the Choice router where you see a vertical black line:
++
+image:mruntime-choice-add-route.png[add+route+option]
+
+=== Add Additional Routes by Editing the XML
+
+You can also add additional route options to a Choice router by modifying the XML config file
```

Comment:
Add missing : at the end.
_You can also add additional route options to a Choice router by modifying the XML config file:_